stubs/ext-random.phpstub

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/**
+ * PHP 8.2 introduces a new PHP extension named "random".
+ * @see https://github.com/php/php-src/blob/master/ext/random/random.stub.php
+ * @see https://php.watch/versions/8.2/ext-random
+ */
+
+define('MT_RAND_MT19937', 0);
+
+define('MT_RAND_PHP', 1);
+
+namespace {
```

Comment:
these functions and consts from above are existing ones (in PHP 8.1 and older) and moved to ext-random. Not sure what to do with stubs in this case

Reply:
Do we need those stubbed at all? Callmap should be enough for those. I skimmed the RFC, what I understood is that everything is in an extension but the extension is always installed and can't be disabled so it's basically part of the core.

Reply:
@orklah 
this is a good question. ext-random should be a part of the code, but you can potentially recompile PHP without it (as far as I understand it in case of this ext). What do you need in such cases - just add to the Callmap?